install_config/cluster_metrics.adoc

```diff
@@ -753,3 +753,291 @@ by performing the following steps:
+openshift_prometheus_pvc_access_modes: [ReadWriteOnce]
+openshift_prometheus_pvc_pv_selector: {}
+
+openshift_prometheus_alertmanager_storage_type: pvc
+openshift_prometheus_alertmanager_pvc_name: prometheus-alertmanager
+openshift_prometheus_alertmanager_pvc_size: 10G
+openshift_prometheus_alertmanager_pvc_access_modes: [ReadWriteOnce]
+openshift_prometheus_alertmanager_pvc_pv_selector: {}
+openshift_prometheus_cpu_limit: null
+openshift_prometheus_memory_limit: null
+openshift_prometheus_cpu_requests: null
+openshift_prometheus_memory_requests: null
+openshift_prometheus_alertmanager_cpu_limit: null
+openshift_prometheus_alertmanager_memory_limit: null
+openshift_prometheus_alertmanager_cpu_requests: null
+openshift_prometheus_alertmanager_memory_requests: null
+openshift_prometheus_alertbuffer_cpu_limit: null
+openshift_prometheus_alertbuffer_memory_limit: null
+openshift_prometheus_alertbuffer_cpu_requests: null
+openshift_prometheus_alertbuffer_memory_requests: null
+openshift_prometheus_oauth_proxy_cpu_limit: null
+openshift_prometheus_oauth_proxy_memory_limit: null
+openshift_prometheus_oauth_proxy_cpu_requests: null
+openshift_prometheus_oauth_proxy_memory_requests: null
+----
+
+*Prometheus Storage Related Variables*
+
+With each Prometheus component--including Prometheus,
+Alertmanager, alert-buffer, and oauth-proxy--you can set the PV claim by setting
+corresponding role variable, for example:
+
+----
+openshift_prometheus_storage_type: pvc
+openshift_prometheus_alertmanager_pvc_name: alertmanager
+openshift_prometheus_alertbuffer_pvc_size: 10G
+openshift_prometheus_pvc_access_modes: [ReadWriteOnce]
+----
+
+*Prometheus Alert Rules File Variable*
+
+You can add an external file with alert rules by setting the path to an additional
+rules variable:
+
+----
+openshift_prometheus_additional_rules_file: <PATH>
+----
+
+The file content should be in Prometheus alert rules format. The following
+example sets a rule to send an alert when one of the cluster nodes is down:
+
+----
+groups:
+- name: example-rules
+  interval: 30s # defaults to global interval
+  rules:
+  - alert: Node Down
+    expr: up{job="kubernetes-nodes"} == 0
+    annotations:
+      miqTarget: "ContainerNode"
+      severity: "HIGH"
+      message: "{{ '{{' }}{{ '$labels.instance' }}{{ '}}' }} is down"
+----
+
+*Prometheus Variables to Control Resource Limits*
+
+With each Prometheus component--including Prometheus, Alertmanager,
+alert-buffer, and oauth-proxy--you can specify CPU, memory limits, and requests
+by setting the corresponding role variable, for example:
+
+----
+openshift_prometheus_alertmanager_limits_memory: 1Gi
+openshift_prometheus_oath_proxy_requests_cpu: 100
+----
+
+For more detailed information, see link:https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_prometheus#openshift-prometheus[OpenShift Prometheus].
+
+[NOTE]
+====
+Metrics can be gathered from the `http://${POD_IP}:7575/metrics` endpoint.
```

Comment:
What metrics? metrics on Prometius, or metrics on my pod/application? We should explain what we're talking about here better.

Reply:
"Metrics can be gathered from the `http://${POD_IP}:7575/metrics` endpoint."
If this paragraph is talking about expose Metric component metrics in Prometheus format, we should tell customer to install Metrics first.

Reply:
I pulled this in from the following Trello card, and can remove it or add to it. What do you think?

https://trello.com/c/Sdk1UryT/572-8-expose-metric-component-metrics-in-prometheus-format-epic-ois-met-exphawkular